### DISCUSSION TITLE
How to disable auto-focus on WindPress fields when editing a Gutenberg block?

### DISCUSSION THREAD

**Discussion by cuongthachHQ** [wind-press/windpress#7] [General]
Auto focus is convenient when designing a new page, but it becomes problematic when editing content. When you select one or more blocks in List view, it automatically focuses on the WordPress field and even opens the suggestion class, which disrupts the editing process.

Is there any way to disable this feature by default, or could I request an option to disable it in the WordPress settings?

![image](https://github.com/user-attachments/assets/3d754a66-ee81-4f70-bdef-714ba5896e24)

**Comment by nguyenquanghoang** [comment]
+1 Totally agree, It is extremely annoying when we edit content

**Comment by suasgn** [comment]
Thank you for pointing this out! We've identified the issue and will fix it in the upcoming release. You should no longer experience the automatic focus when editing the Block. We appreciate your feedback!
